<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>measurementDialog</class>
 <widget class="QDialog" name="measurementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool Wear Analyzer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Drag and drop the reference image:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Drag and drop mask for this image:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="imgDropLabel" name="maskImg" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="imgDropLabel" name="referenceImg" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Drag and drop extracted line:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="imgDropLabel" name="lineImg" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="3">
      <widget class="QWidget" name="plot" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>imgDropLabel</class>
   <extends>QWidget</extends>
   <header>imageedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
